app/src/main/java/com/rage/clamber/Fragments/ActionBarFragment.java

```diff
@@ -0,0 +1,86 @@
+package com.rage.clamber.Fragments;
+
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.rage.clamber.Activities.HomePage;
+import com.rage.clamber.Activities.ProjectsActivity;
+import com.rage.clamber.Activities.UserInfoActivity;
+import com.rage.clamber.Activities.WallActivity;
+import com.rage.clamber.R;
+
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class ActionBarFragment extends Fragment {
+
+
+    public ActionBarFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Standard method to create a new ActionBar Fragment. This method is called
+     * within the various activities to set the Fragment to the top of their layout.
+     * @return returns the ActionBarFragment.
+     */
+    public static ActionBarFragment newInstance () {
+        ActionBarFragment fragment = new ActionBarFragment();
+        Bundle args = new Bundle();
```

Comment:
do you need to create the args bundle, even if the fragment doesn't need any args?